packages/select/test/suggestTests.tsx

```diff
@@ -210,6 +211,23 @@ describe("Suggest", () => {
         }
     });
 
+    describe("Controlled Mode", () => {
```

Comment:
please also test that interactions do not get into a weird intermediate state. see other test suites.